frontends/election-manager/src/components/import_cvrfiles_modal.tsx

```diff
@@ -97,7 +97,9 @@ export function ImportCvrFilesModal({ onClose }: Props): JSX.Element {
       fileData,
       election
     );
-    await saveCastVoteRecordFiles(newCastVoteRecordFiles);
+    await addCastVoteRecordFile(
```

Comment:
is your plan to remove the newCastVoteRecordFiles part of this? And this addCastVoteRecordFile function will return an error if there is an error or if its a duplicate file?

Reply:
Yes, eventually the `addCastVoteRecordFile` will return a value that can be used to replace `newCastVoteRecordFiles`.